psalm-baseline.xml

```diff
@@ -1851,12 +1837,6 @@
     </UninitializedProperty>
   </file>
   <file src="lib/Doctrine/ORM/Query/Exec/MultiTableUpdateExecutor.php">
-    <InvalidReturnStatement>
```

Comment:
You shouldn't touch sections for files you didn't modify, that's for sure. I suppose that was done automatically, so maybe you don't have the correct version of Psalm installed? Make sure to run `composer update`.

Reply:
Yeah so basically I'd have to go over the change of the baseline file and make sure it only fixes stuff in-scope of my pr, gotcha. I'll sort it out.